docs/API.md

```
@@ -54,9 +54,38 @@ Some endpoints may return data directly without the `ok` wrapper:
+}
+```
+**Example with zod-validation-error:**
+```json
+{
+  "ok": false,
+  "error": "Validation error: Name must contain at least 3 character(s) at 'name'",
+  "details": { ... }
 }
 ```
+**Note:** The `details` field is optional and typically populated for 400 Bad Request (Validation) errors.
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)